scene/3d/collision_object_3d.cpp

```diff
@@ -110,6 +111,41 @@ void CollisionObject3D::_update_pickable() {
 	}
 }
 
+void CollisionObject3D::_update_debug_shapes() {
+	for (Set<uint32_t>::Element *shapedata_idx = debug_shapes_to_update.front(); shapedata_idx; shapedata_idx = shapedata_idx->next()) {
+		if (shapes.has(shapedata_idx->get())) {
+			ShapeData shapedata = shapes[shapedata_idx->get()];
+			for (int i = 0; i < shapedata.shapes.size(); i++) {
+				ShapeData::ShapeBase s = shapedata.shapes[i];
```

Comment:
Same thing here, it should be `ShapeData::ShapeBase &s`

Reply:
Unnecessary, as I write it back below.

Reply:
Just to clarify, here you could just access the ShapeBase by reference with `ShapeData::ShapeBase &s = shapedata.shapes.write[i];` and remove the line `shapedata.shapes.write[i] = s;` instead of making a copy twice.

Reply:
I wasn't aware it is possible. Good to know!